spec/swagger/definitions/hand_coded_paths_v2.yaml

```diff
@@ -110,3 +110,62 @@
           application/vnd.api+json:
             schema:
               $ref: post_people_responses.yaml#/422
+/people/{person_id}:
+  patch:
```

Comment:
can you mark this as deprecated/do not use?

Reply:
👍